docs/design.adoc

```diff
@@ -303,19 +293,19 @@ The governance included in the system design follows two principles:
 
 * All governance should abide by a time delay, giving users time to respond
   to changes in the system.
-* The governance role should be assignable to a credibly neutral third party or 
+* The governance role should be assignable to a credibly neutral third party or
   eventual decentralization, such as the community multisig.
 
 .Governable parameters
```

Comment:
This section still needs to be cleared out. We still have here:
- Deposit liquidation notifier reward percentage
- Deposit liquidated notifier reward percentage

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/146/commits/98ae98c3462967491810499def1dc1bfbbff4a45